modules/pricer/src/main/java/com/opengamma/strata/pricer/fxopt/SmileDeltaParameters.java

```diff
@@ -338,6 +338,30 @@ public DoubleArray impliedStrikesDerivativeToExpiry(double forward) {
     return DoubleArray.ofUnsafe(dStrikedTime);
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Calculates the derivatives of the implied strikes to forward.
+   *
+   * @param forward  the forward
+   * @return the derivatives of the implied strikes
+   */
+  public DoubleArray impliedStrikesDerivativeToForward(double forward) {
+    int nbDelta = delta.size();
+    double[] dStrikedTime = new double[2 * nbDelta + 1];
```

Comment:
Should the variable name here be `dStrikedForward`? There's some other references to time in method as well.